src/app/core/models/types/analog.ts

```diff
@@ -24,6 +32,10 @@ export const DefaultAnalogComponent: { [C in AnalogComponent as C["kind"]]: Comp
     "AnalogNode": (id) => ({ ...DefaultComponent(id), kind: "AnalogNode"                 }),
     "Ground":     (id) => ({ ...DefaultComponent(id), kind: "Ground"                     }),
     "Resistor":   (id) => ({ ...DefaultComponent(id), kind: "Resistor", resistance: 1000 }),
+    "Inductor":   (id) => ({ ...DefaultComponent(id), kind: "Inductor", inductance: 10 }),
+    "CurrentSource": (id) => ({ ...DefaultComponent(id), kind: "CurrentSource", amperage: 1 }),
```

Comment:
keep these all vertically aligned at
`(id) =>`
and similarly, keep the end of the lines vertically aligned with the `}),`